src/main/java/spoon/reflect/visitor/PrinterHelper.java

```diff
@@ -144,9 +144,7 @@ private void writeTabsInternal() {
 				if (env != null) {
 					indentationSize = env.getTabulationSize();
 				}
-				for (int j = 0; j < indentationSize; j++) {
-					sbf.append(' ');
-				}
+				sbf.append(Character.toString(' ').repeat(indentationSize));
```

Comment:
This refactoring is definitely better than `for-loop`. I would suggest we can further shorten this to:
```java
sb.append("\s".repeat(indentationSize));
```

I am unsure if `\s` escape sequence works like `\t` does.

Reply:
I merge @monperrus suggestions, but yours looks fine too.